packages/@aws-cdk/aws-s3/test/bucket.test.ts

```diff
@@ -724,7 +724,19 @@ describe('bucket', () => {
         bucketName: 'mybucket',
       });
 
-      expect(bucket.bucketWebsiteUrl).toEqual(`http://mybucket.s3-website.us-east-2.${stack.urlSuffix}`);
+      expect(bucket.bucketWebsiteUrl).toEqual('http://mybucket.s3-website.us-east-2.amazonaws.com');
+    });
+
+    test('old bucketWebsiteUrl format for specific region', () => {
```

Comment:
Nice! Can you duplicate this set of tests were we pass the region to `fromBucketAttributes` instead of the Stack?

Reply:
Sure, that makes sense 👍

Reply:
Added some tests, but had to remove `@deprecated` because I got `DeprecationError` when testing `BucketAttributes.bucketWebsiteNewUrlFormat`:/